Dockerfile

```
@@ -26,6 +26,9 @@ RUN        bash /tmp/env-config.sh
 # Calypso development.
 COPY       . /calypso/
 RUN        npx lerna bootstrap --ci
+WORKDIR    node_modules/node-sass
+RUN        npm run install
```

Rather than three commands to change and change back, let's just `RUN` one:

```suggestion
RUN        cd node_modules/node-sass && npm run install
```